src/attack_flow_builder/public/settings_macos.json

```diff
@@ -0,0 +1,72 @@
+{
+    "file": {
+        "image_export": {
+            "padding": 30
+        }
+    },
+    "edit": {
+        "clone_offset": [20, 20]
+    },
+    "view": {
+        "diagram": {
+            "display_grid": true,
+            "display_shadows": true,
+            "display_debug_mode": false,
+            "render_high_quality": true,
+            "disable_shadows_at": 0.7
+        }
+    },
+    "hotkeys": {
+        "file": {
+            "new_file": "Meta+N",
```

Comment:
On Chrome 116 / MacOS, this shortcut opens a new browser window!

Reply:
I've switched it back to `Ctrl+N` so that it's vaguely familiar. We might consider removing the keyboard shortcut altogether now that plans for multi-page document editing have been scrapped. I doubt this shortcut will be used much.